CHANGELOG.md

```diff
@@ -43,6 +43,8 @@
 
 ##### Bug Fixes
 
+* Fix `ignores_case_statements` typo in doc string for `cyclomatic_complexity`  [Jeff Blagdon](https://github.com/jefflovejapan)
```

Comment:
formatting for this doesn't follow the guidelines in [CONTRIBUTING.md](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes)